zerver/actions/streams.py

```diff
@@ -133,6 +133,67 @@ def do_deactivate_stream(
     )
 
 
+def bulk_delete_cache_keys(message_ids_to_clear: List[int]) -> None:
+    while len(message_ids_to_clear) > 0:
+        batch = message_ids_to_clear[0:5000]
+
+        keys_to_delete = [to_dict_cache_key_id(message_id) for message_id in batch]
+        cache_delete_many(keys_to_delete)
+
+        message_ids_to_clear = message_ids_to_clear[5000:]
+
+
+def merge_streams(realm: Realm, stream_to_keep: Stream, stream_to_destroy: Stream) -> List[int]:
```

Comment:
The return type of this isn't a `List` -- that is, it's not variable length, and it's only happenstance that they're all the same type.  We know this is always going to be exactly three elements, so it should be a `Tuple[int, int, int]`.

Reply:
DONE - thx! for your patience, I'm obviously new to typed python...